packages/react-aria-components/src/TableLayout.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {TableLayout as BaseTableLayout, TableLayoutProps} from '@react-stately/layout';
+import {LayoutOptionsDelegate} from './Virtualizer';
+import {TableColumnResizeStateContext} from './Table';
+import {useContext, useMemo} from 'react';
+
+export class TableLayout<T> extends BaseTableLayout<T> implements LayoutOptionsDelegate<TableLayoutProps> {
```

Comment:
Another thing I just realized is that this exposes options like `loaderHeight` that actually don't do anything due to how TableLayout implements its own `buildCollection`. I've gotten rid of it from stately's TableLayout but I'll double check if there are any other options that need to be removed/aren't applicable to RAC TableLayout